src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,134 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+
+public class RegistrationServiceImplTest {
+    private static final String PASSWORD_TEST = "123123";
+    private static final String LOGIN_TEST = "MyTestLogin";
+    private static final int AGE_TEST = 34;
```

Comment:
Check your test method according to common mistake 
[Add tests for all possible User’s parameters (null login/password/age, user under 18/18/over 18 years old, negative age, and so on…)](https://mate-academy.github.io/jv-program-common-mistakes/java-core/junit/registration-validation)

Reply:
It's better to use age test 18 for valid case. 
Look comment above